components/MintButton/MintButton.tsx

```diff
@@ -58,15 +60,18 @@ const MintButton: FC<MintButtonProps> = ({ twitterHandle }) => {
     const file = dataURLtoFile(dataUri, "a.png")
     const ipfsUrl = await storeBlob(file)
 
-    const receipt = (await axios.get(`/api/mint?imageUri=${ipfsUrl}`)) as any
+    const generatedBy = transformTwitterHandle(twitterHandle)
+    const receipt = (await axios.get(
+      `/api/mint?imageUri=${ipfsUrl}&name=${generated}&description=${generated} generated by ${generatedBy}`,
+    )) as any
```

Comment:
you can prob refactor this to be more readable
```
const receipt = await axios.get('/api/mint',{params{imageUri:ipfsUrl, name: generated, description:`${generated} generated by ${generatedBy}`}})```

Reply:
updated ✅ 
<img width="574" alt="Screenshot 2022-12-23 at 10 22 38 AM" src="https://user-images.githubusercontent.com/23249402/209367140-a8389323-b3f1-4902-ae2b-959a5882ce48.png">